src/client/components/features/OffreDeStage/Consulter/ConsulterOffreDeStage.tsx

```diff
@@ -47,11 +48,7 @@ export function ConsulterOffreDeStage({ offreDeStage }: ConsulterOffreDeStagePro
 		}
 		tags.push(dureeCategorisee(offreDeStage.dureeEnJour || 0));
 		if (offreDeStage.dateDeDebutMin) {
-			tags.push(
-				offreDeStage.dateDeDebutMax && offreDeStage.dateDeDebutMin !== offreDeStage.dateDeDebutMax
-					? `Débute entre le : ${new Date(offreDeStage.dateDeDebutMin).toLocaleDateString()} et ${new Date(offreDeStage.dateDeDebutMax).toLocaleDateString()}`
-					: `Débute le : ${new Date(offreDeStage.dateDeDebutMin).toLocaleDateString()}`,
-			);
+			tags.push(getDateStageFormatted(offreDeStage.dateDeDebutMin, offreDeStage.dateDeDebutMax));
```

Comment:
```suggestion
			tags.push(formatDateStage(offreDeStage.dateDeDebutMin, offreDeStage.dateDeDebutMax));
```
La fonction ne permet pas de récupérer la date, elle permet de formatter une date passée en paramètre